### DISCUSSION TITLE
generate files报错，有人知道大概啥问题吗

### DISCUSSION THREAD

**Discussion by yaodong5063** [umijs/umi#9226] [General]
info - generate files
fatal - Error: [BABEL] C:\work\polyfill.ts: @babel/helper-compilation-targets: 'false' is not a valid value for 'targets.edge'. (While processing: "C:\work\node_modules\https://github.com/umijs\bundler-utils\compiled\babel\preset-env.js")

**Comment by xiaohuoni** [comment]
配置？

**Comment by undefined996** [comment]
所以最后解决了么

**Comment by fz6m** [comment]
如有问题，请提供一个最小复现。
